lib/velum/salt_orchestration.rb

```diff
@@ -0,0 +1,22 @@
+# frozen_string_literal: true
+require "velum/salt_api"
+
+module Velum
+  # This class represents a salt orchestration
+  class SaltOrchestration
+    include SaltApi
+
+    attr_accessor :orchestration
+
+    # Initializes a new salt orchestration backend matching orchestration
+    def initialize(orchestration:)
+      @orchestration = orchestration
+    end
+
+    # Return information for this orchestration
+    def info
+      res = perform_request(endpoint: "/jobs/#{@orchestration.jid}", method: "get")
```

Comment:
i think if you have defined an accessor, you dont need to reference class variables anymore, meaning `s/@orchestration.jid/orchestration.jid/g`

Reply:
Oh, yeah, I think is a matter of taste, I try to avoid one extra method call if I have direct access to the variable.